Popular_Models_Guide/StableDiffusion/docs/client_application.md

```diff
@@ -0,0 +1,48 @@
+<!--
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Client Command Line Options
```

Comment:
I don't think this separate doc is needed, and it will be hard to maintain along with updates to the client script. 

Maybe we can just add something like "To customize the options used with the client script, see `python3 client.py -h`" to the section in the main README that uses the client script?

Reply:
@fpetrini15 - what's your take? I kind of like having a reference for each CLI with additional examples. Reference vs tutorial. At the moment - no examples :) - so could definitely remove and add back later.

@rmccorm4  main thing I wanted to show was the concurrency / batch size options - Maybe a section in main readme or an intro paragraph here?

the model configuration file was another one - as a future PR - I think I would just add comments to the file.

Reply:
I second that this is not needed. Maybe in the `[Popular_Models_Guide/StableDiffusion/README.md](https://github.com/triton-inference-server/tutorials/pull/82/files#diff-ffe1f530b57aee0dd3d7c1c98b053fdceb10f60eee18c68b363a930429a55928)` we can say something like `python3 client.py --help` has all the command line options and please refer to it?